src/API/Request/NumericBoost.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace GroupByInc\API\Request;
+
+use JMS\Serializer\Annotation as JMS;
+
+class NumericBoost
+{
+  /**
+   * @var string
+   * @JMS\Type("string")
+   */
+  private $name;
+
+  /**
+   * @var boolean
+   * @JMS\Type("boolean")
+   */
+  private $inverted;
+
+  /**
+   * @var double
+   * @JMS\Type("double")
+   */
+  private $strength = 1.0;
+
+  /**
+   * @return string The name of this Numeric Boost
+   */
+  public function getName()
+  {
+    return $this->name;
+  }
+
+  /**
+   * Set the name of this Numeric Boost
```

Comment:
@grrilla Suggestion; remove this comment as it describes the functionality(or what) of the method as opposed to the why. The code is clear enough to know this without the comment. Please check the whole file for occurrences of this.

Reply:
Done